kolibri/plugins/management/assets/src/state/store.js

```diff
@@ -8,9 +8,9 @@ const initialState = {
   pageState: {},
   facility: undefined,
   users: [], // this should be inside page state
+  roles: [],
```

Comment:
If you were the one that added the above comment, looks like both `roles` and `users` would be better suited for the `pagestate` object?